libs/application/templates/aosh/work-accident-notification/src/fields/Components/CausesAndEffects.tsx

```
@@ -0,0 +1,197 @@
+        <MultiSelectDropdownController
+          onAnswerChange={onChange}
+          groups={activityGroups}
+          items={activites}
+          pickedValue={pickedValue}
+          {...props}
+        />
+      </Box>
+      {errors && getErrorViaPath(errors, answerId) && (
+        <Box paddingTop={1}>
+          <ErrorMessage
+            children={formatMessage(
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)